ush/hpssarch_gen.sh

```diff
@@ -234,12 +234,13 @@ if [ $type = "gdas" ]; then
   if [ -s $ROTDIR/${dirpath}${head}radstat ]; then
      echo  "${dirname}${head}radstat                 " >>gdas.txt
   fi
-  for fstep in prep anal gldas fcst vrfy radmon minmon oznmon; do
-   if [ -s $ROTDIR/logs/${CDATE}/gdas${fstep}.log ]; then
-     echo  "./logs/${CDATE}/gdas${fstep}.log         " >>gdas.txt
-   fi
-  done
-  echo  "./logs/${CDATE}/gdaspost*.log               " >>gdas.txt
+#  for fstep in prep anal gldas fcst vrfy radmon minmon oznmon; do
+#   if [ -s $ROTDIR/logs/${CDATE}/gdas${fstep}.log ]; then
+#     echo  "./logs/${CDATE}/gdas${fstep}.log         " >>gdas.txt
+#   fi
+#  done
+#  echo  "./logs/${CDATE}/gdaspost*.log               " >>gdas.txt
```

Comment:
can you please explain what is the intent behind this?
Are logs not supposed to go to archive?

Reply:
This script wired in the log file using rocoto workflow format. It is modified to include all log file in the log directory, so ecflow log file can also be included in backup set.

Reply:
Does `ecflow` run in development mode for all users?
Does `ecflow` jobs archival jobs call `arch.sh` or `earc.sh`? 
I have not seen those jobs in the `ecf/`.
So how are these being accessed/called?

Reply:
perhaps I am missing something but just looking at the logic of the script we will no longer be archiving gdasprep.log, gdasanal.log etc ? Looks like we are only archiving gdaspost*.log ? Or perhaps I am missing the bigger picture ?

Reply:
Line 237-242 hardwired to only backup rocoto workflow log file. ecflow workflow does not use the same log file name. Therefore, removed hardwired name and use the line 243 and 378 to take in all log file found in log directory.